lib/request_headers_logger/json_formatter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module RequestHeadersLogger
+  class JsonFormatter
+    def call(severity, timestamp, progname, msg)
+      json = { level: severity, timestamp: timestamp.to_s, message: msg.strip }
+      json = json.merge(progname: progname.to_s) unless progname.nil?
+
+      json.merge(RequestHeadersLogger.tags || {}).to_json + "\n"
```

Comment:
This is what I kinda don't really like without having a much better solution. But this is an implicit dep on ActiveSupport::TaggedLogger. and I don't think this is really needed like this for Json logging.

Reply:
No this is not a dep on ActiveSupport::TaggedLogger.  

`RequestHeadersLogger.tags ` is just the store items after filtering the whitelisted ones
see [here](https://github.com/fidor/request_headers_logger/blob/master/lib/request_headers_logger.rb#L14)